### DISCUSSION TITLE
They rolled a huge bug from this commit

### DISCUSSION THREAD

**Discussion by krive001** [qtile/qtile#3027] [Q&A]
They rolled a huge bug from this commit
commit number: ~57f0963fbd63f1c32b383412bc9bbc093b38dc22~
diff:
[bugstart.txt](https://github.com/qtile/qtile/files/7534240/bugstart.txt)

**Comment by m-col** [comment]
Sorry it's not clear what the bug is - could you add a bit more detail? Thanks.

**Comment by krive001** [comment]
Insert this into the config.py file.
```
for x in qtile.widgets_map:
    logger.warning(f"map: {x}")

~/.local/share/qtile/qtile.log
```
output `cat ~/.local/share/qtile/qtile.log`
Reload two.

Good: `WARNING libqtile config.py:<module>():L538 map: systray`
Error: ` WARNING libqtile config.py:<module>():L538 map: <no name>`

**Comment by elParaguayo** [comment]
That's not a huge bug. That's a log message that you've inserted.

What's the actual problem that you're having?

Yes, Systray widget currently shows as `<no name>` but I'm not sure why that matters. The widget still works as it did.

EDIT: And please use the issue template when reporting bugs.

**Comment by krive001** [comment]
Yes, but how do you draw a window, a non-existent widget?

**Reply by elParaguayo** [reply]
Take a step back. So far, you've told us absolutely nothing about what you're trying to do, what you expect to happen and what actually happens (this is why we have the template for reporting issues).

Please answer those questions and we can then see if there's actually a bug or not.

I assume this is related to you trying to fix the Systray issue we've discussed previously.

**Reply by elParaguayo** [reply]
I'm pretty sure you're doing something wrong if you're trying to access the widget through `widgets_map`.

**Comment by elParaguayo** [comment]
#3028 should fix the name issue but I'd still really like to understand what you're trying to do!
